src/models/searchservice/response/QueryRulesActionsData.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Data returned from the Answers query rules system.
+ *
+ * @public
+ */
+export interface QueryRulesActionsData {
+  /**
+   * The unique identifier for this query rule.
+   */
+  key: string,
```

Comment:
can this be truly any string? it would be nice if product had a list of the exact values we could get here, if possible. a little more structure would be nice for data too

Reply:
Yes, this can be any string. The unique identifier for the query rule is decided by the customer, so it's arbitrary.

Reply:
As for `data`, this is also dictated by the customer. I believe they can specify arbitrary JSON to be returned. It's a bit like the `rawData` case we already have in Core. It's on them to really know the structure of `data`.